### ISSUE TITLE
Help with reproducing the numbers in the paper, on MUSDBHQ.

### ISSUE THREAD

**Issue by maswang32** [gladia-research-group/cocola#2]
Thanks for the amazing work!

We would like to use your metric to evaluate music generation models. As a start, I am trying to use your model on some of the datasets in the paper, including MUSDB18-HQ. I am using the Dataset provided in the [codebase ](https://github.com/gladia-research-group/cocola/blob/main/data/musdb_contrastive_preprocessed.py) for Musdb, on the test split, and computing COCOLA scores as described in the README. However, my scores are closer to ~30 (instead of ~57). Here is a minimal example reproducing the issue:

```
import numpy as np
import torch
from contrastive_model import constants
from contrastive_model.contrastive_model import CoCola
from feature_extraction.feature_extraction import CoColaFeatureExtractor
from data.musdb_contrastive_preprocessed import MusdbContrastivePreprocessed
from feature_extraction.feature_extraction import CoColaFeatureExtractor

# Load Model
device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
model = CoCola.load_from_checkpoint("cocola/cocola_hp_v1.ckpt").to(device)
feature_extractor = CoColaFeatureExtractor().to(device)
model.eval()
model.set_embedding_mode(constants.EmbeddingMode.BOTH) # Harmonic and Percussive score

# Load Test Dataset
dataset = MusdbContrastivePreprocessed(
            root_dir="~/musdb_contrastive",
            download=False,
            preprocess=True,
            split="test",
            chunk_duration=5,
            target_sample_rate=16000,
            generate_submixtures=True,
            device="cpu",
            preprocess_transform=None,
            runtime_transform=None)

scores = []

for datapoint in dataset:
    with torch.no_grad():
        x = datapoint['anchor']
        y = datapoint['positive']
        features_x = feature_extractor(x)
        features_y = feature_extractor(y)
        score = model.score(features_x.to(device), features_y.to(device))
        scores.append(score.item())
        print(score)
```

I get similar scores when excluding "Vocal" and "Other", as discussed in the paper.

I get similar scores when I use MoisesDB or Coco Chorales.

Please let me know if you have insight for what may be causing the discrepancy!

Best,
Mason Wang

**Comment by rubenciranni** [comment]
It turns out the link provided in the previous REAME file was pointing to the wrong checkpoint. We have updated the README with the correct checkpoint in the last commit.

Best,
Ruben
